src/rest.rs

```diff
@@ -862,6 +864,38 @@ fn handle_request(
             json_response(query.estimate_fee_map(), TTL_SHORT)
         }
 
+        (&Method::GET, Some(&"mw-outputs"), Some(scan_master_priv_key), Some(spend_pub_keys_list), None, None) => {
```

Comment:
Not a huge deal, but I'm not in love with the idea of passing around a private key. If an alternative solution presents itself at some point, it'd be worth looking into.

Reply:
There are no ways to avoid this. You've got to have scan private key to identify outputs.